tests/unit/test_merger_arxiv2arxiv.py

```diff
@@ -1289,9 +1289,13 @@ def test_merging_dois_field():
     expected_merged = {
         'dois': [
             {
-                'material': 'publication',
+                'material': 'preprint',
                 'source': 'nowhere',
                 'value': '10.1023/A:1026654312961'
+            },
+            {
+                'material': 'publication',
+                'value': '10.1023/A:1026654312961'
```

Comment:
I'd add a test that does actually match the dois also, as we just lost it (there was none that did not though).

Reply:
:+1: actually you made me realize the new comparator is wrong when we have two DOIs from the same publisher, because of errata typically. I will need to fix that.